### PR TITLE
Update Footer.js [ Improved Footer Section in Mobile Design]

### LINKED ISSUE #9313: [FEATURE] Improvement: footer section on mobile view
### Description

Footer section doesn't look good on mobile view.

### Screenshots

IMG_20231004_121114

### Do you want to work on this issue?

Yes

### Additional information

I'm thinking to provide `flex-direction=column` so the text `100% Open Source on GitHub` and icons align column wise.

### PR DESCRIPTION
**Pull request by PradeepG-07**
## Fixes Issue
Improved Footer Section in Mobile Design
Closes #9313 

## Changes proposed
I have wrapped the links in a separate div and then for the paragraph "100% Open source on Github" added the whitespace-nowrap property.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
In Desktop view
Screenshot 2023-10-09 195431
In Mobile view
Screenshot 2023-10-09 195322

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -206,22 +206,24 @@ export default function Footer() {
           </div>
         </div>
         <div className="mt-8 border-t border-white/10 pt-8 flex flex-col md:flex-row   items-center justify-between">
-          <div className="flex pb-4 md:pb-0 items-center justify-center space-x-6 md:order-2">
-            <p className=" text-xs leading-5 text-primary-low-high  md:mt-0">
+          <div className="flex pb-4 md:pb-0 items-center justify-center space-x-6 md:order-2 flex-wrap">
```
Comment (amandamartin-dev) [inline components/Footer.js:209]: Hello! Thank you for working on this.  It is looking nice, but the icons are not centered on mobile under the text.  This appears to be due to the space-x-6 on this div creating a space that is not needed on mobile, but remains needed on desktop in the current configuration..  There are a few ways you can approach addressing this, but take a look and see if you can ensure the icons are centered on mobile.

left space

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

File: components/Footer.js (hunk 1 above)
Comment (PradeepG-07) [reply components/Footer.js]: Hey @amandamartin-dev  the changes are made as you said. Please review it incase of any query please tell me. Thank you.

Comment (PradeepG-07) [reply components/Footer.js]: Screenshot 2023-10-10 100338
Screenshot 2023-10-10 100356

**Comment by PradeepG-07** [comment]
Please merge this pull request I want to work on new issues. Thank you.

**Comment by amandamartin-dev** [comment]
> Please merge this pull request I want to work on new issues. Thank you.

we require 2 maintainers to review on any issues other than profile updates.  Someone will have. a look and merge when they are able.  Thank you for your patience

**Comment by PradeepG-07** [comment]
Please help me. 🙏

**Comment by eddiejaoude** [comment]
Reviewing now

**Comment by eddiejaoude** [comment]
Great collaboration 💪  thank you both, merging now

**Pull request merged**